s4e-backend/src/main/java/pl/cyfronet/s4e/controller/InvitationController.java

```diff
@@ -0,0 +1,82 @@
+@RequiredArgsConstructor
+@Tag(name = "invitation", description = "Invite by email API")
+public class InvitationController {
+    private final InvitationService invitationService;
+    private final ApplicationEventPublisher eventPublisher;
+
+    @Operation(summary = "Send invitation email to user")
+    @ApiResponses({
+            @ApiResponse(responseCode = "200", description = "Invitation was send"),
+            @ApiResponse(responseCode = "400", description = "Incorrect request", content = @Content),
+            @ApiResponse(responseCode = "401", description = "Unauthenticated", content = @Content),
+            @ApiResponse(responseCode = "403", description = "Forbidden", content = @Content),
+            @ApiResponse(responseCode = "404", description = "Not found", content = @Content)
+    })
+    @PostMapping(value = "/institutions/{institution}/invitations", consumes = APPLICATION_JSON_VALUE)
+    public void send(
+            @RequestBody @Valid CreateInvitationRequest request,
+            @PathVariable("institution") String institutionSlug
+    ) throws Exception {
+        val token = invitationService.createInvitationFrom(request.getEmail(), institutionSlug);
+        val invitationEvent = new OnSendInvitationEvent(token, LocaleContextHolder.getLocale());
+        eventPublisher.publishEvent(invitationEvent);
+    }
+
+    @Operation(summary = "Confirm invitation")
+    @ApiResponses({
+            @ApiResponse(responseCode = "200", description = "Invitation have been confirmed"),
+            @ApiResponse(responseCode = "400", description = "Incorrect request", content = @Content),
+            @ApiResponse(responseCode = "401", description = "Unauthenticated", content = @Content),
+            @ApiResponse(responseCode = "404", description = "Not found", content = @Content)
+    })
+    @PostMapping(value = "/invitations/{token}/confirm", consumes = APPLICATION_JSON_VALUE)
+    public BasicInstitutionResponse confirm(@PathVariable String token) throws NotFoundException {
+        invitationService.confirmInvitation(token);
+        val institution = invitationService.findInstitutionBy(token, BasicInstitutionResponse.class)
+                .orElseThrow(() -> new NotFoundException("Institution couldn't be found"));;
+        val confirmInvitationEvent = new OnConfirmInvitationEvent(token, LocaleContextHolder.getLocale());
+        eventPublisher.publishEvent(confirmInvitationEvent);
+
+        return institution;
+    }
+
+    @Operation(summary = "Reject invitation")
+    @ApiResponses({
+            @ApiResponse(responseCode = "200", description = "Invitation have been rejected"),
+            @ApiResponse(responseCode = "400", description = "Incorrect request", content = @Content),
+            @ApiResponse(responseCode = "404", description = "Not found", content = @Content)
+    })
+    @PutMapping(value = "/invitations/{token}/reject", consumes = APPLICATION_JSON_VALUE)
```

Comment:
`consumes = APPLICATION_JSON_VALUE` is not necessary as you don't accept a request body here.